.github/actions/send-slack-notification/action.yml

```diff
@@ -24,7 +24,7 @@ runs:
   steps:
     - name: Validate token
       if: inputs.token == ''
-      uses: actions/github-script@v3
+      uses: actions/github-script@v7
       with:
         script: |
           core.setFailed('Slack token was not provided!')
```

Comment:
@atchertchian @wojciech-piotrowiak 

is this worth having instead of the usual:

```
echo 'Slack token was not provided!'
exit 1
```

?

Reply:
`core.setFailed` is documented as "Used as a shortcut for `::error` and `exit 1`" (https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-creating-an-annotation-for-an-error), I don't mind using one or the other, the only difference with your suggestion is that we don't have `::error` so no annotation but not sure it really matters (?)

Reply:
In this particular case, I agree. Anyway probably not something that should be changed as part of a dependabot PR  anyway :)